responsive-web-design/css-typography/index04.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Nutrition Label</title>
+  </head>
+  <body>
+    <h1>Nutrition Facts</h1>
+    <p>8 servings per container</p>
+    <p>Serving size 2/3 cup (55g)</p>
+  </body>
+</html>
+
+<!-- 
+It's important to pick a typeface (or font) that has many different weights. 
+Add the following link to the head element, just under the title element: 
+`<link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700,800" rel="stylesheet">`
```

Comment:
Explain what this is doing. Breakdown each property `href` and `rel` and also discuss where users can pick fonts (like explain what google fonts is). Additionally, explain what a 'font weight' is.

Reply:
The cat photo app @beaucarnes is working on will fall before this one, and should cover things like `<link>`, `href`, and `rel`, so I just briefly explained what font weights are. I could come back in and review all of the things you mentioned, though.

Reply:
Nope, I was totally wrong about this. There will be no CSS portion to the cat photo app. Sorry about that! I'll try to split this challenge up a bit to explain the different parts like you suggested.